src/sentry/api/serializers/rest_framework/dashboard.py

```diff
@@ -115,6 +115,9 @@ class DashboardWidgetQuerySerializer(CamelSnakeSerializer):
         child=serializers.CharField(), required=False, allow_null=True
     )
     columns = serializers.ListField(child=serializers.CharField(), required=False, allow_null=True)
+    field_aliases = serializers.ListField(
+        child=serializers.CharField(), required=False, allow_null=True
```

Comment:
```suggestion
        child=serializers.CharField(allow_blank=True), required=False, allow_null=True
```
Realized that this won't let us set empty strings if a field doesn't have an alias. Can we add a test to cover this case as well?